apps/web/app/[lang]/studies/[id]/components/StudyPreviewContent.tsx

```diff
@@ -0,0 +1,124 @@
+import { P2, Card, H4, Button, HorizontalDivider } from 'ui/core';
+import { useTranslations } from 'intl';
+import { ImageWithFallback } from '@components/utils/ImageWithFallback';
+import { AgeToText, GenderToText } from '../helpers';
+import { StudyPreviewContentProps } from '../Study.types';
+
+export const StudyPreviewContent: React.FC<StudyPreviewContentProps> = ({
+  study,
+}) => {
+  const t = useTranslations('studies.studyDetail');
+  const tActions = useTranslations('actions');
+  const tStudyTypes = useTranslations('studies.studyDetail.types');
+  const ageRequirements = AgeToText(
+    study.eligibility.minimum_age,
+    study.eligibility.maximum_age,
+  );
+
+  return (
+    <Card className="w-full max-h-min flex flex-col items-center justify-center !m-0 !p-4 gap-6">
+      <div className="w-full flex flex-col gap-4">
```

Comment:
```suggestion
      <div className="flex flex-col gap-4">
```

If the parent is w-full and flex, and the children is also flex, the default width will be 100% already

Reply:
Removing the w-full class makes content not fill all the container. I think it is related to the sibling button element also specifying full width, forcing the parent div to be wider than this div.  
However, also removing w-full from button makes it shrink to fit-content, so I think there is no way around setting both to w-full so they completely fill the parent grid cell